packages/trader/src/Stores/Modules/Trading/__tests__/trade-store.spec.ts

```diff
@@ -0,0 +1,318 @@
+    {
+        allow_forward_starting: 1,
+        display_name: 'Volatility 100 (1s) Index',
+        display_order: 3,
+        exchange_is_open: 1,
+        is_trading_suspended: 0,
+        market: 'synthetic_index',
+        market_display_name: 'Derived',
+        pip: 0.01,
+        subgroup: 'synthetics',
+        subgroup_display_name: 'Synthetics',
+        submarket: 'random_index',
+        submarket_display_name: 'Continuous Indices',
+        symbol,
+        symbol_type: 'stockindex',
+    },
+];
+
+jest.mock('@deriv/shared', () => {
+    const commonRiseFallProperties = {
+        barrier_category: 'euro_atm',
+        barriers: 0,
+        exchange_name: 'RANDOM',
+        market: 'synthetic_index',
+        submarket: 'random_index',
+        underlying_symbol: symbol,
+    };
+    const commonNonEqualProperties = {
+        contract_category: 'callput',
+        contract_category_display: 'Up/Down',
+    };
+    const commonEqualProperties = {
+        contract_category: 'callputequal',
+        contract_category_display: 'Rise/Fall Equal',
+    };
+    const commonRiseProperties = {
+        contract_display: 'Higher',
+        sentiment: 'up',
+    };
+    const commonFallProperties = {
+        contract_display: 'Lower',
+        sentiment: 'down',
+    };
+    const forwardStartingOptions = [
+        {
+            close: '1708991999',
+            date: '1708905600',
+            open: '1708905600',
+        },
+        {
+            close: '1709078399',
+            date: '1708992000',
+            open: '1708992000',
+        },
+        {
+            close: '1709164799',
+            date: '1709078400',
+            open: '1709078400',
+        },
+    ];
+    const commonForwardStartingProperties = {
+        expiry_type: 'intraday',
+        forward_starting_options: forwardStartingOptions,
+        max_contract_duration: '1d',
+        min_contract_duration: '2m',
+        start_type: 'forward',
+    };
+    const commonDailyExpiryProperties = {
+        expiry_type: 'daily',
+        max_contract_duration: '365d',
+        min_contract_duration: '1d',
+        start_type: 'spot',
+    };
+    const commonIntradayExpiryProperties = {
+        expiry_type: 'intraday',
+        max_contract_duration: '1d',
+        min_contract_duration: '15s',
+        start_type: 'spot',
+    };
+    const commonTickExpiryProperties = {
+        expiry_type: 'tick',
+        max_contract_duration: '10t',
+        min_contract_duration: '1t',
+        start_type: 'spot',
+    };
+    const contractsForResponse = {
+        contracts_for: {
+            available: [
+                {
+                    ...commonRiseFallProperties,
+                    ...commonNonEqualProperties,
+                    ...commonRiseProperties,
+                    ...commonDailyExpiryProperties,
+                    contract_type: 'CALL',
+                },
+                {
+                    ...commonRiseFallProperties,
```

Comment:
It seems like there's a lot of repetitions in these test cases, could you please reduce it into a single object?